share/picongpu/tests/metadataFromLaserWakefield/picongpu-metadata.json.reference

```diff
@@ -0,0 +1,22 @@
+{
+  "incidentField": {
+    "XMax": {},
+    "XMin": {},
+    "YMax": {},
+    "YMin": {
+      "polarisation": {
```

Comment:
Offline discussed: We can have many lasers per direction therefore it is important that each laser is getting a name

Reply:
More precisely: Having a name is content which is later added by the domain experts. The important thing to make this correct as an infrastructure is that each entry "XMin", ... maps to a list of objects (which might or might not have a name then).